README.md

```diff
@@ -1 +1,170 @@
-# inheritance
\ No newline at end of file
+# Grocery Store
+
+Let's simulate a grocery store system! We want to be able to keep track of the orders that folks make, both online and physically in our grocery store.
+
+This project will allow you to explore object-oriented design as well as a few other new topics.
+
+## Baseline Setup
+
+1. This is an individual, stage 1 project.
+1. Fork the project master.
+1. Clone the forked repo: `$ git clone [YOUR FORKED REPO URL]`
+1. `cd` into the dir created `$ cd grocery-store`
+1. Run `git remote -v` to verify the folder you are in corresponds to the fork you have created.
```

Comment:
Since this is a stage 1 project, maybe an example of the correct and incorrect response to `git remote -v`.